x-pack/solutions/observability/packages/kbn-streams-schema/src/models/stream_config/ingest_stream_config.ts

```diff
@@ -10,10 +10,12 @@ import { processingDefinitionSchema, streamChildSchema } from '../common';
 
 export const ingestStreamConfigDefinitonSchema = z
   .object({
-    ingest: z.object({
-      processing: z.array(processingDefinitionSchema),
-      routing: z.array(streamChildSchema),
-    }),
+    ingest: z
+      .object({
+        processing: z.array(processingDefinitionSchema),
+        routing: z.array(streamChildSchema),
+      })
+      .strict(),
```

Comment:
added `strict()` on the inner object otherwise classic streams defining a lifecycle would pass zod validation and we'd have to validate downstream. this can be removed once we also support lifecycle update for classic streams